proto/minder/v1/minder.proto

```diff
@@ -1244,6 +1244,8 @@ message GitHubAppProviderConfig {
   string app_id = 3;
   // UserID is the GitHub App User ID
   int64 user_id = 4;
+  // PrivateKeyPath is the path to the GitHub App Private Key
+  string private_key_path = 5;
```

Comment:
Do we want to make this configurable? Should this be an implementation detail?

Reply:
Similar to the app ID and endpoint, if we have we're talking to multiple GitHubs, private and public, then we'll need multiple GitHub Apps and each app will have it's own private key. The private key is generated from GitHub, we don't choose our own, so we couldn't reuse the same key for multiple apps.